test/extensions/filters/http/common/fuzz/uber_filter.h

```diff
@@ -60,6 +63,19 @@ class UberFilterFuzzer {
     ON_CALL(factory_context_.admin_, removeHandler(_)).WillByDefault(testing::Return(true));
   }
 
+  static void addProtoDescriptor(absl::string_view filter_name, Protobuf::Message* message) {
+    if (filter_name.find("grpc_json_transcoder") != absl::string_view::npos) {
+      envoy::extensions::filters::http::grpc_json_transcoder::v3::GrpcJsonTranscoder& config =
+          dynamic_cast<
+              envoy::extensions::filters::http::grpc_json_transcoder::v3::GrpcJsonTranscoder&>(
+              *message);
+      config.clear_services();
+      config.add_services("bookstore.Bookstore");
+      config.set_proto_descriptor(
+          absl::StrCat(std::getenv("TEST_SRCDIR"), "/test/proto/bookstore.descriptor"));
```

Comment:
Best to use `TestEnvironment` to form these paths. It has convenience wrapper for this env stuff.

Reply:
In my first commit I had, and got a stack trace crash from RLocation when using `TestEnvironment::runfilesPath(_)`. I know runfiles was not nullptr (since the assert guards that). I'll try again.